superset/models/core.py

```diff
@@ -913,7 +913,7 @@ def get_extra(self):
 
     def get_table(self, table_name, schema=None):
         extra = self.get_extra()
-        meta = MetaData(**extra.get('metadata_params', {}))
+        meta = MetaData(extra.get('metadata_params', {}))
```

Comment:
Looking at [the docs](https://docs.sqlalchemy.org/en/latest/core/metadata.html#sqlalchemy.schema.MetaData), I think this is what we want. This will simply pass the dictionary as the `bind` argument, which is incorrect.

The real problem here is that `connection_args` should not be in the `metadata_params` dict: https://superset.incubator.apache.org/installation.html#deeper-sqlalchemy-integration

It should contain only keys that are part of `MetaData.__init__`:

```python
__init__(bind=None, reflect=False, schema=None, quote_schema=None, naming_convention={'ix': 'ix_%(column_0_label)s'}, info=None)
```

Reply:
You are right! My bad. I copied the setting from the database configuration in our production but did not check whether the customized configuration is in a correct format. 

I should be more careful. Thanks for it!